pkg/util/shell/shell.go

```diff
@@ -88,40 +89,32 @@ func GenerateUsageHint(userShell, cmdLine string) string {
 	return fmt.Sprintf("%s Run this command to configure your shell:\n%s %s\n", comment, comment, cmd)
 }
 
-func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suffix, delimiter string) {
+func GetPrefixSuffixDelimiterForSet(userShell string) (prefix, delimiter, suffix, pathSuffix string) {
 	switch userShell {
 	case "fish":
 		prefix = "set -gx "
-		suffix = "\";\n"
 		delimiter = " \""
-		if pathVar {
-			suffix = "\" $PATH;\n"
-		}
```

Comment:
@svenstaro @praveenkumar Why is this change required? i.e. changes to `pathvar`

Reply:
@LalatenduMohanty @svenstaro introduced a new member `PathSuffix` in struct `ShellConfig` and managing the current form of building path having `PATH` with it. Nice one! :+1: